packages/paste-website/src/component-examples/page-templates/ObjectsListExamples.ts

```diff
@@ -0,0 +1,291 @@
+              <ScreenReaderOnly>label</ScreenReaderOnly>
+            </Checkbox>
+          </DataGridCell>
+          <DataGridCell>
+            <Anchor href="#">Anchor</Anchor>
+          </DataGridCell>
+          <DataGridCell>Content</DataGridCell>
+          <DataGridCell>Content</DataGridCell>
+          <DataGridCell textAlign="right">Content</DataGridCell>
+        </DataGridRow>
+        <DataGridRow>
+          <DataGridCell>
+            <Checkbox>
+              <ScreenReaderOnly>label</ScreenReaderOnly>
+            </Checkbox>
+          </DataGridCell>
+          <DataGridCell>
+            <Anchor href="#">Anchor</Anchor>
+          </DataGridCell>
+          <DataGridCell>Content</DataGridCell>
+          <DataGridCell>Content</DataGridCell>
+          <DataGridCell textAlign="right">Content</DataGridCell>
+        </DataGridRow>
+      </DataGridBody>
+    </DataGrid>
+  </Box>
+</Box>`.trim();
+
+export const DefaultObjectsListExample = `
+<Box
+  paddingX="space100"
+  paddingTop="space130"
+  paddingBottom="space160"
+  display="flex"
+  flexDirection="column"
+  rowGap="space130"
+>
+  <Box display="flex" columnGap="space30" justifyContent="space-between" alignItems="flex-start">
+    <Box maxWidth="size70">
+      <Box display="flex" rowGap="space30" marginBottom="space30">
+        <Breadcrumb aria-label={useUID()}>
+          <BreadcrumbItem href="#">Phone numbers</BreadcrumbItem>
+          <BreadcrumbItem href="#">Manage</BreadcrumbItem>
+        </Breadcrumb>
+      </Box>
+      <Heading as="h4" variant="heading10">
+        Verified caller IDs
+      </Heading>
+      <Paragraph marginBottom="space0">
+        Verify a number that you own to use it as a caller ID or as the “To” number for outbound calls and messages.
+        The phone numbers you buy from Twilio or port to Twilio can always be used as caller IDs.
+      </Paragraph>
+    </Box>
+    <Box marginBottom="space30" marginTop="space80" whiteSpace="nowrap">
+      <Button variant="primary">Add a new caller ID</Button>
+    </Box>
+  </Box>
+  <Box>
+    <Box maxWidth="size70" display="flex" flexDirection="column" rowGap="space50" marginBottom="space90">
+      <Box display="flex" columnGap="space80" alignItems="flex-end">
+        <Box maxWidth="size90" display="flex" columnGap="space50">
+          <Box width="100%">
+            <Label htmlFor={input1}>Phone number</Label>
+            <Input type="text" id={input1} />
+          </Box>
+          <Box width="100%">
+            <Label htmlFor={input2}>Friendly name</Label>
+            <Input type="text" id={input2} />
+          </Box>
+        </Box>
+        <Box display="flex" width="size20" justifyContent="flex-end">
+          <ButtonGroup>
+            <Button variant="secondary">
+              <FilterIcon decorative />
+              Apply
+            </Button>
+            <Button variant="secondary">Clear</Button>
+          </ButtonGroup>
+        </Box>
+      </Box>
+    </Box>
```

Comment:
fix: urgh, not sure why I didn't catch this during focus time.

The spacing between the form fields and ButtonGroup here should be the same as what it is in the "Full Objects List Example"